ci-operator/step-registry/ipi/conf/proxy/ipi-conf-proxy-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
The  file name - `ipi-conf-proxy-commands.sh` sounds like it is for editing install-config purpose, but actually we are trying to create `proxy_conf.sh`, I propose "ci-operator/step-registry/proxy/conf/proxy-conf-commands.sh"

Reply:
agree, `ipi/conf` is for _clusters_, proxy server should be out of it, and it's not only for IPI, but also UPI and other operators' e2e tests, `ci-operator/step-registry/proxy/conf` is good to me,  but I think something like `ci-operator/step-registry/proxy/generateproxyconf` is more clear to users.